packages/commons-server/src/libs/response-rules-interpreter.ts

```diff
@@ -175,7 +180,7 @@ export class ResponseRulesInterpreter {
       return value.includes(rule.value);
     }
 
-    return String(value) === String(rule.value);
+    return String(value) === String(this.parseValue(rule.value));
```

Comment:
I think I would parse the `rule.value` earlier, like on line ~165, so the reex or array include would also check on the parsed value.

Reply:
Oh yes, that's a good point!